src/ActionModal/utils/MessageConstructor.js

```diff
@@ -46,3 +61,167 @@ export const getMultiMessage = async (context, messages) => {
     }
   }
 }
+
+// Bank
+export function MsgSend(
```

Comment:
Better export these functions from the library instead?

Reply:
Agreed, but they are not currently exported from the library. I can do this at a later time.